### DISCUSSION TITLE
Combine transfers between bank/card asset accounts and topup fees

### DISCUSSION THREAD

**Discussion by theRolento** [firefly-iii/firefly-iii#8920] [Firefly III] [answered]
What is the correct workflow or how do you combine transfers between asset accounts?

There are always double transfers when importing data from two banks with data importer (GoCardless or file) and after mapping them.

For example:
1) Import from Revolut bank -> mapped incoming transfer from MainBank into Revolut
2) Import from MainBank -> mapped outcoming transfer from MainBank into Revolut

Simply keep one and delete the other? (that's what i'm doing)
Isn't there a way to merge/combine them manually into one?

Another question is: how to manage fees when topping up debit/prepaid cards?

And the only logic selectable type for debit/prepaid cards is "default asset account", a suggestion could be to put two more types for them.

Thank you

**Comment by floli** [comment, marked as answer]
I was asking a similar question here: https://github.com/orgs/firefly-iii/discussions/8849

> Simply keep one and delete the other? (that's what i'm doing)

tl;dr Just delete one.
